### PR TITLE
Introduce new bundles with swagger core upgrade

### PR DESCRIPTION
**Pull request by YasasRangika**
### Purpose
This PR will create new versions for the below three dependency bundles.

***swagger-parser/2.0.20.wso2v2***
This version contains swagger core version 1.6.9 to fix the known vulnerabilities

***swagger-inflector/1.0.16.wso2v2***
This version also contains swagger core version 1.6.9 to fix the known vulnerabilities

***openapi-generator/4.3.1.wso2v1.1***
This version of openapi-generator includes guava version range extension to fix the OSGI issues after upgrading swagger core versions to 1.6.9 in API Manager-3.2.0

### CODE REVIEW

File: openapi-generator/4.3.1.wso2v1.1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (hasuniea) [inline openapi-generator/4.3.1.wso2v1.1/pom.xml:3]: should be 2023

File: swagger-inflector/1.0.16.wso2v2/pom.xml [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (hasuniea) [inline swagger-inflector/1.0.16.wso2v2/pom.xml:3]: here as well. Change the year.

File: swagger-parser/2.0.20.wso2v2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (hasuniea) [inline swagger-parser/2.0.20.wso2v2/pom.xml:3]: should be 2023

File: openapi-generator/4.3.1.wso2v1.1/pom.xml (hunk 1 above)
Comment (YasasRangika) [reply openapi-generator/4.3.1.wso2v1.1/pom.xml]: fixed with https://github.com/wso2/orbit/pull/938/commits/bbc74c4bd027d2c656b1752e44de28ad0e70548f

File: swagger-inflector/1.0.16.wso2v2/pom.xml (hunk 2 above)
Comment (YasasRangika) [reply swagger-inflector/1.0.16.wso2v2/pom.xml]: fixed with https://github.com/wso2/orbit/pull/938/commits/bbc74c4bd027d2c656b1752e44de28ad0e70548f

File: swagger-parser/2.0.20.wso2v2/pom.xml (hunk 3 above)
Comment (YasasRangika) [reply swagger-parser/2.0.20.wso2v2/pom.xml]: fixed with https://github.com/wso2/orbit/pull/938/commits/bbc74c4bd027d2c656b1752e44de28ad0e70548f

**Review by hasuniea — APPROVED** [review]

**Pull request merged**